src/patient-registration/input/custom-input/autosuggest.css

```diff
@@ -0,0 +1,64 @@
+
+.react-autosuggest__container {
+    position: relative;
+  }
```

Comment:
This is probably all fine for now, but I've asked whether there's a common pattern for autocomplete with Carbon: https://github.com/carbon-design-system/carbon/issues/3513#issuecomment-700965898